Quadriga/src/main/java/edu/asu/spring/quadriga/domain/impl/ConceptpowerReply.java

```diff
@@ -96,7 +99,29 @@
     public void setConceptEntry(List<ConceptpowerReply.ConceptEntry> entries) {
         this.conceptEntry = entries;
     }
+    
+    @XmlElement(required = true, namespace="http://www.digitalhps.org/")
+    protected List<ConceptpowerReply.AlternativeIds> alternativeIds;
```

Comment:
This does not have to be a list, does it? There should only be one element alternativeIds, right?